### DISCUSSION TITLE
Added toolkit MediaElement to my solution and it worked until I disabled Hot Reload

### DISCUSSION THREAD

**Discussion by sqlguy6212** [CommunityToolkit/Maui#944] [New Feature Discussions]
When I disabled Hot Reload it crashes on InitCompontent (MainPage).  Can't say much more than that other than if I uncomment out the line :
<toolkit:MediaElement Grid.Row="2" x:Name="theMediaView" ShouldAutoPlay="True" ShouldKeepScreenOn="True" ShouldShowPlaybackControls="True" Volume="100"></toolkit:MediaElement>

It crashes and tells you nothing useful.

**Comment by sqlguy6212** [comment]
Seems to be the Volume property.  If I add the line again and add the properties one at a time it fails when I add Volume.  It seems that the 100 is the problem and that if I replace that with 1 it works.  Need to find docs about the properties.

**Reply by bijington** [reply]
This sounds like an issue rather than a discussion about a new feature. Can you please open an issue from this section?

https://github.com/CommunityToolkit/Maui/issues/new/choose

**Comment by sqlguy6212** [comment]
Well previous comment is gone so lets try again.  The problem (I added the properties back one at a time was with Volume 100.  This caused the crash and replacing it with Volume 1 fixed it (at least on emulator).  On my phone (Samsung) there is no volume while playing video (from web).

**Comment by sqlguy6212** [comment]
A note about this site.  I have added several comments and when I come back they do not show.  As soon as I add this one the others will show.

**Reply by bijington** [reply]
That's because comments like this result in starting a thread so you only end up seeing the comments that relate to the original comment

**Reply by sqlguy6212** [reply]
Ok did the last thing which is see if other apps have sound on my phone, and no they don't.  If I put bluetooth on the video has sound so only problem is the value of Volume

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
